fixtures/large-error/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "large-error"
+version = "0.26.1"
+authors = ["Alexander Cyon <alex.cyon@gmail.com>"]
```

Comment:
Let me know if you want this to say `["Firefox Sync Team <sync-team@mozilla.com>"]` instead....

Reply:
I'm fine with giving you the credit. Note however that if you choose it can be omitted entirely.